subsys/net/lib/nrf_cloud/src/nrf_cloud_fota.c

```diff
@@ -455,16 +521,18 @@ static void http_fota_handler(const struct fota_download_evt *evt)
 	case FOTA_DOWNLOAD_EVT_FINISHED:
 		/* MCUBOOT: download finished, update job status and reboot */
 		current_fota.status = NRF_FOTA_IN_PROGRESS;
-		save_validate_status(current_fota.id, current_fota.type,
-				NRF_FOTA_VALIDATE_PENDING);
+		save_validate_status(current_fota.info.id,
```

Comment:
I'm missing something basic here... where is the BLE's firmware downloaded, and where is it stored temporarily?

Reply:
thats up to the application... this fota module doesn't know how or when the BLE device needs to be updated.